.github/workflows/push.yml

```
@@ -236,23 +247,56 @@ jobs:
         run: |
-          ./scripts/ci/10_prepare-docker-deps.sh
+          if aws ecr describe-images --repository-name $DEPS_CACHE_REPO --image-ids imageTag=${{ steps.cache-tag.outputs.tag }} &>/dev/null; then
+            echo "cache_exists=true" >> $GITHUB_OUTPUT
+          else
+            echo "cache_exists=false" >> $GITHUB_OUTPUT
+          fi
+        continue-on-error: true
+
+      - name: Build and push deps layer
+        if: steps.check-deps-cache.outputs.cache_exists != 'true'
+        uses: docker/build-push-action@v4
```

actionlint (1.7.4)

the runner of "docker/build-push-action@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)